client/src/components/App.js

```diff
@@ -2,7 +2,7 @@ import React, {Fragment} from 'react'
 import Navigation from './Navigation'
 import {Route} from 'react-router-dom'
 import {Switch} from 'react-router'
-import Verejne from './Verejne'
+import Verejne from './Verejne/Verejne'
```

Comment:
Not that I'd had an idea for a better name, but it bothers me a bit that this is the only slovak name in the modules naming.